src/UiPatternInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Drupal\ui_patterns;
+
+/**
+ * Interface UiPatternInterface.
+ *
+ * @package Drupal\ui_patterns
+ */
+interface UiPatternInterface {
```

Comment:
Same here, is this something generic or it belongs to the `ui_patterns_config` sub-module?